org.civicrm.bootstrapcivicrm/scss/civicrm/custom-civicrm.scss

```diff
@@ -7,10 +7,12 @@
 @import '../bootstrap/vendor/font-awesome/variables';
 @import '../bootstrap/vendor/font-awesome/path';
 @import '../bootstrap/overrides/style/*';
+@import 'variables';
```

Comment:
Shouldn't this be the very first `@import`, to make sure these variables do not get overridden?
```scss
@import 'variables';
@import '../bootstrap/overrides/variables';
@import '../../../org.civicrm.bootstrap/scss/overrides/variables';
// ...
```